packages/apps/fortune/exchange-oracle/server/src/modules/job/job.service.spec.ts

```diff
@@ -342,13 +363,18 @@ describe('JobService', () => {
         fundAmount: 100,
       };
 
+      const assignment = {
+        jobId: 1,
+        workerAddress: workerAddress,
+        status: AssignmentStatus.ACTIVE,
+      };
+
```

Comment:
```suggestion
      assignment.status = AssignmentStatus.ACTIVE;
```
Same here. This way, we reuse the code and keep it cleaner